assets/src/edit-story/components/panels/preset/karma/stylePresets.karma.js

```diff
@@ -35,6 +35,9 @@ describe('Panel: Style Presets', () => {
   };
 
   beforeEach(async () => {
+    // Make sure panels are starting at initial state on every test
+    localStorage.clear();
```

Comment:
Hm, this should probably go in some global reset. @swissspidy, suggestions?

Reply:
Yea let me know here. I'm not the most familiar with the Karma tests, but think this was causing the main issue

Reply:
Yeah probably. `karma/fixture/init.js` has similar resets already.

Reply:
Can be done in a follow-up ticket / PR too, FWIW